src/__experimental__/components/simple-color-picker/simple-color-picker.spec.js

```diff
@@ -435,4 +436,16 @@ describe('SimpleColorPicker', () => {
       expect(console.error).toHaveBeenCalledWith(expected); // eslint-disable-line no-console
     });
   });
+
+  it('the required prop is passed to the input', () => {
+    const wrapper = render(mount, { required: true });
+
+    const inputs = wrapper.find('input');
+    inputs.forEach((input) => {
+      expect(input.prop('required')).toBe(true);
+    });
+
+    const fieldset = wrapper.find(Fieldset);
+    expect(fieldset.prop('isRequired')).toBe(true);
+  });
```

Comment:
two tests in one case

Reply:
Updated test description because the setup is the same.